documentation/contributing/styleguide.adoc

```diff
@@ -1,6 +1,35 @@
 [[style-guide]]
 = Style guidelines
 
+[[general-styles]]
+== Adding code and configuration examples
+
+The Strimzi documentation contains many code and configuration examples.
+Examples are a useful way of demonstrating a process.
+
+If you want to add example code and configuration to your contribution, use the following format in an asciidoc code block.
+
+.Configuration example
+[source,subs="+quotes"]
+------
+[source,yaml,subs="+quotes,attributes"] <1>
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    replicas: *3* # \<1> <2>
+    # ... <3>
```

Comment:
How will the callouts inside callouts looks in the documentation? Did you tried if it renders reasonably well with Asciidoc?

Reply:
Rendered fine when I checked locally.